jormungandr-integration-tests/src/common/jcli_wrapper/jcli_transaction_wrapper/jcli_transaction_commands.rs

```diff
@@ -86,6 +86,30 @@ impl TransactionCommands {
         command
     }
 
+    pub fn get_finalize_with_fee_command(
+        &self,
+        address: &str,
+        fee_constant: &i32,
+        fee_coefficient: &i32,
+        fee_certificate: &i32,
+        staging_file: &PathBuf,
```

Comment:
This should be `&Path`, or better a generic type `P: AsRef<Path>` to also accept strings.